### PR TITLE
`mqclient` Updates [minor]

### PR DESCRIPTION
**Pull request by ric-evans**
Adds `EWMS_PILOT_PREFETCH`, which is `0` by default (previously `1`)

### CODE REVIEW

**Comment by ric-evans** [comment]
TODO: remove prefetch

File: ewms_pilot/pilot.py [hunk 1]
```diff
@@ -27,7 +27,7 @@
 
 _DEFAULT_TIMEOUT_INCOMING = 1  # second
 _DEFAULT_TIMEOUT_OUTGOING = 1  # second
-_DEFAULT_PREFETCH = 1
+_DEFAULT_PREFETCH = 0
```
Comment (dsschult) [inline ewms_pilot/pilot.py:30]: Is there still a need for this variable, with the change below to `ENV.EWMS_PILOT_PREFETCH`?

Comment (ric-evans) [reply ewms_pilot/pilot.py:30]: Nope :smile: I guess flake8 didn't look at this scenario

Comment (ric-evans) [reply ewms_pilot/pilot.py:30]: ah, it was used in one other place

Comment (ric-evans) [reply ewms_pilot/pilot.py:30]: https://github.com/Observation-Management-Service/ewms-pilot/pull/42

**Pull request merged**